src/test/scala/mesosphere/marathon/state/GroupManagerTest.scala

```diff
@@ -87,7 +87,80 @@ class GroupManagerTest extends MarathonActorSupport with MockitoSugar with Match
     ))
     val update = manager(minServicePort = 10, maxServicePort = 20).assignDynamicServicePorts(Group.empty, group)
     update.transitiveApps.filter(_.hasDynamicPort) should be ('empty)
-    update.transitiveApps.flatMap(_.portNumbers.filter(x => x >= 10 && x <= 20)) should have size 2
+    update.transitiveApps.flatMap(_.hostPorts.flatten.filter(x => x >= 10 && x <= 20)) should have size 2
```

Comment:
why? your suggestion is not consistent with prior behavior. I changed `portNumbers` to `hostPorts.flatten` here to also consider the host ports specified for containers (vs the host ports specified in `ports`). using `servicePorts` instead is a much different semantic.

we could **add** a check for `servicePorts` but they would not all fall into a range of `[10,20)` since there's a manually specified servicePort of `10555`